RNGNewAuraNotifier/Core/Json/JsonUpdateService.cs

```diff
@@ -16,8 +17,9 @@ internal class JsonUpdateService(string owner, string repo)
     /// <returns>masterブランチのAuras.jsonが取得され、最新バージョンであればローカルに保存されたときに完了するタスク</returns>
     public async Task FetchMasterJsonAsync()
     {
+        ConfigData configData = AppConfig.Instance;
         var url = new Uri($"https://raw.githubusercontent.com/{_owner}/{_repo}/master/{_repo}/Resources/Auras.json");
-        var saveDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), "RNGNewAuraNotifier", "Aura.json");
+        var saveDir = Path.Combine(configData.AuraJsonDir, "Auras.json");
```

Comment:
ディレクトリに対するパスではないので、Dirではない方がいいかもです。

```suggestion
        var saveFilePath = Path.Combine(configData.AuraJsonDir, "Auras.json");
```